File: api/api.web/Model/SessionWebModel.fs
namespace Apex.Api.Web.Model

open System
open System.ComponentModel.DataAnnotations

type SessionDto = {
    [<Required>]
    id : int

    [<Required>]
    user : UserDto

    [<Required>]
    token : string

    [<Required>]
    createdOn : DateTime

    [<Required>]
    expiresOn : DateTime
}

[<CLIMutable>]
type LoginRequestDto = {
    [<Required>]
    username : string

    [<Required>]
    [<StringLength(20, MinimumLength = 6)>]
    password : string
}

File: api/api.web.model/SessionWebModel.cs
using System;

namespace Apex.Api.Web.Model
{
    public class Session
    {
        public int Id { get; set; }
        public User User { get; set; }
        public string Token { get; set; }
        public DateTime CreatedOn { get; set; }
        public DateTime ExpiresOn { get; set; }
    }

    public class LoginRequest
    {
        public string UserName { get; set; }
        public string Password { get; set; }
    }
}